packages/web/src/features/fileTree/components/fileTreeItemComponent.tsx

```
@@ -0,0 +1,108 @@
+        >
+            <div
+                className="flex flex-row gap-1 cursor-pointer w-4 h-4 flex-shrink-0"
+            >
+                {isCollapseChevronVisible && (
+                    <>
+                        {isCollapsed ? (
+                            <ChevronRightIcon className="w-4 h-4 flex-shrink-0" />
+                        ) : (
+                            <ChevronDownIcon className="w-4 h-4 flex-shrink-0" />
+                        )}
+                    </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)